tests/Microsoft.Identity.Test.Unit/ManagedIdentityTests/AzureArcTests.cs

```diff
@@ -17,8 +17,6 @@ namespace Microsoft.Identity.Test.Unit.ManagedIdentityTests
     [DeploymentItem("Resources\\ManagedIdentityAzureArcSecret.txt")]
     public class AzureArcTests
     {
-        private const string Endpoint = "http://localhost:40342/metadata/identity/oauth2/token";
-        private const string Resource = "https://management.azure.com";
         private const string AzureArc = "Azure Arc";
 
         [TestInitialize]
```

Comment:
Please inherit from `TestBase` and delete the test initialize. TestBase will add a test intilize and reset things for you.